### ISSUE TITLE
Search page should provide an indication of the term the user entered.

### ISSUE THREAD

**Issue by danfox01** [baltimorecounty/BCPL-assets#600]
Something like, "Showing results for **{term}**"

This is low priority and does not have to be in place for launch.

**Comment by martypowell** [comment]
@sgrg1 if you take a look at the page in this state https://www.bcpl.info/search-results.html?term=books

The search term is "books" and should show up in the circled section below.

Showing results for "books"

![image](https://user-images.githubusercontent.com/1933400/55158761-083fe600-5136-11e9-8b97-e59743827b14.png)

**Comment by sgrg1** [comment]
added search item to display on a page

**Comment by martypowell** [comment]
@sgrg1 not seeing these changes in dev?

**Comment by sgrg1** [comment]
@martypowell  changes has been pushed to dev . Ready for PR

**Comment by martypowell** [comment]
@sgrg1 please create the pull request, once you guys are happy with the changes, it will be submitted for review, once approved, we will send to @danfox01 team to test. Once they are happy we will complete the pull request.

**Comment by tmccoy529** [comment]
@danfox01 this is ready for testing as well.

**Comment by danfox01** [comment]
@jdomasky, can you please test this one as well? You'll need to use a direct search link in order to hit dev, since the search query box will take you to prod. Eg: https://dev.bcpl.info/search-results.html?term=things

**Comment by jdomasky** [comment]
@danfox01 I'll try to test this today, within the next hour. I'll report my results here.

**Comment by jdomasky** [comment]
@sgrg1 @tmccoy529 @martypowell @danfox01 Hi guys, testing didn't go so well earlier today. I found two issues:

1. **The search term is displayed with the URL encoding intact.** Try this search:
https://dev.bcpl.info/search-results.html?term=service%20animal
Currently in dev, the term is displayed as: **service%20animal**
The desired format of display is: **service animal**

2. The second is a minor issue. On the search results page, when the browser width is reduced, the search term wraps to a second line long before it should.
Because we have a generous amount of empty space in the enclosing DIV, the term shouldn't wrap until the Search the Catalog button is much closer.
This can be reproduced by dragging the size of a desktop browser window narrower until the search term wraps to a second line.
The desired behavior is for the label and search term to remain on the same line until the Search the Catalog button is close enough to trigger a natural line wrap.

**Comment by tmccoy529** [comment]
@sgrg1 @tmccoy529 @martypowell @danfox01 @jdomasky I have added a decode to the text to keep that pesky %20 from being added to the div. As far as the second issue this is something that is currently in production so that needs to be separated out as a new issue.

![image](https://user-images.githubusercontent.com/40430541/56133780-4a04c500-5f5b-11e9-9c27-340b9e0b7b61.png)

**Comment by jdomasky** [comment]
@tmccoy529 Looks good! I've confirmed in dev that the search term is now being displayed without the URL encoding.
I'll let @danfox01 decide whether we should pursue the second issue.

**Comment by martypowell** [comment]
> @sgrg1 @tmccoy529 @martypowell @danfox01 @jdomasky I have added a decode to the text to keep that pesky %20 from being added to the div. As far as the second issue this is something that is currently in production so that needs to be separated out as a new issue.
> 
> ![image](https://user-images.githubusercontent.com/40430541/56133780-4a04c500-5f5b-11e9-9c27-340b9e0b7b61.png)

@sgrg1 @tmccoy529 the second item reported by jason is valid for this issue. When we were working on the code, I just threw "<em>" without any sort of knowledge. I just figured we would want to style the term somehow. I would suggest trying to replicate the behavior of em with  span tag and a class.

**Comment by tmccoy529** [comment]
@martypowell This is already in production this way. So its unrelated to this particular issue. When resizing production the button jumps down the screen to a new line with massive amounts of white space beside it th3e div with text. I'm sorry to push back but we can probably find many other flaws in this section and shouldn't be fixing them coupled with this enhancement. One is a bug already in prod one is an enhancement.

**Comment by danfox01** [comment]
@tmccoy529 You are correct that this is already in production this way, but that is actually the expected behavior. The existing text and button are supposed to become block-level elements at a certain breakpoint, otherwise you end up with something like this:
![FireShot Capture 049 - Search Results - Baltimore County Public Library - dev bcpl info](https://user-images.githubusercontent.com/13049389/56138769-86d5b980-5f65-11e9-960f-8ae8f42d475d.png)

This issue, I believe, is that the new `<em>` tag is also inheriting the block styling from its parent, and it should be inline.

**Comment by tmccoy529** [comment]
@danfox01 @martypowell @jdomasky one of the google css classes was forcing a block display on everything inside of it. This was naughty and has been fixed to ignore span and em tags and instead make them inline. This appears to have fixed the word wrap ugliness.

**Comment by danfox01** [comment]
Thanks! @jdomasky please retest

**Comment by jdomasky** [comment]
OK. Will retest today, probably within the next hour.

**Comment by jdomasky** [comment]
@danfox01 The retest in dev was entirely successful.
